configs/components/gem-prune.rb

```diff
@@ -0,0 +1,38 @@
+component 'gem-prune' do |pkg, settings, platform|
+  pkg.build_requires 'bolt-runtime'
+
+  pkg.add_source('file://resources/rubygems-prune')
+
+  pkg.build do
+    puppet_cachedir = File.join(settings[:privatedir], 'puppet', 'ruby')
+    ruby_cachedir = File.join(settings[:cachedir], 'ruby')
+
+    gem_bins = {
+      settings[:ruby_api] => settings[:host_gem],
+    }
+
+    settings[:additional_rubies]&.each do |rubyver, local_settings|
+      gem_bins[local_settings[:ruby_api]] = local_settings[:host_gem]
+    end
+
+    bolt_ruby_versions = ['2.5.0', '2.7.0']
```

Comment:
We only ship one ruby version for bolt. You can find that in puppet-runtime in the bolt-runtime project definition https://github.com/puppetlabs/puppet-runtime/blob/341e6f0a196c74d38dda93d0b036b16c4207ca34/configs/projects/bolt-runtime.rb#L4